### DISCUSSION TITLE
Submitting PR review times out

### DISCUSSION THREAD

**Discussion by fboja** [community/community#21807] [New to GitHub]
<p>I am attempting to submit a big PR review (100+ comments). When attempting to submit the large review I see the (infamous) unicorn page with a <strong>This page is taking way too long to load</strong>  error message.</p>
<p>Previously I have had no problems submitting large reviews, but this time around I can’t get it working for some reason.</p>
<p>I have tried the following:</p>
<ol>
<li>
<p>Reduced number of comments</p>
</li>
<li>
<p>Submitting through multiple browsers to verify that it is a Github backend issue</p>
</li>
<li>
<p>Verified that the PR <em>can</em> be reviewed (colleague could easily submit a small review without issue)</p>
</li>
</ol>
<p>I have attempted to contact GitHub support, but no response for two days now so wanted to check in here. Any good ideas would be highly appreciated :slight_smile:</p>
<p>PS: The review uses the new multi-line code suggestion feature extensivly, and that is my main suspect right now!</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/fboja">@fboja</a>, I have seen a few people post about this over the last few weeks. My honest suggestion as a workaround while you wait for support to respond is to batch the review into smaller pieces (which I know may not be ideal). </p>

**Comment by fboja** [comment]
<p>I split up my 1 large review into two smaller ones, and that did indeed allow me to submit the review(s). Thanks  for that <a class="mention" href="/logankilpatrick">@logankilpatrick</a>, it fixed my current problem :) </p>
<p>After manually copy/pasting out of the initial review, I could suddenly post the review. Some of the comments I moved had a certain structure. See example below:</p>
<pre><code>1. var x = "abc";
2. if (x == "y")
3. Console.log("yey");
</code></pre>
<ul>
<li>Line 1 had a single comment</li>
<li>Line 1-3 had a multiline comment containing a multiline code suggestion covering line 1-3</li>
</ul>
<p>I could not reproduce this with 100% certainty, but this might be a useful reference for others running into this problem in the future.</p>
